docs/nddata/index.rst

```diff
@@ -92,17 +92,17 @@ Instances are created in the same way::
     >>> ndd
     NDDataRef([1, 2, 3, 4])
 
-But also support arithmetics (:ref:`nddata_arithmetic`) like addition::
+But also support arithmetic (:ref:`nddata_arithmetic`) like addition::
 
     >>> import astropy.units as u
     >>> ndd2 = ndd.add([4, 3, 2, 1] * u.erg / u.s)
     >>> ndd2
     NDDataRef([ 5.,  5.,  5.,  5.])
 
-because these operations have a wide range of options these are not avaible
-using pythons arithmetic operators like ``+``.
+Because these operations have a wide range of options these are not available
+using arithmetic operators like ``+``.
 
-Slicing or indexing (:ref:`nddata_slicing`) is possible (issueing warnings if
+Slicing or indexing (:ref:`nddata_slicing`) is possible (issuing warnings if
```

Comment:
Reminder, the sentence in the paranthesis is wrong. That would require #4271 which will not be part of 1.2 (?)